homeworks/artur.kobyliatsky_Allocene/GamesProject/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.contain {
+  width: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.content {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+  justify-content: center;
+}
+
+.text {
+  width: 575px;
+  height: 478px;
+}
+
+.text h1 {
+  margin-top: 140px;
+  color: #221F1F;
+  font-size: 48px;
+}
+
+.text p {
+  color: #221F1F;
+  font-size: 22px;
```

Comment:
please add a `line-heigh` from the desing 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/8c707cad-d3ef-4ed4-8bfa-ec86f48b4c53)

Reply:
Yes, sorry about that, missed this moment)